pkg/discovery/globalnet/globalnet_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package globalnet
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("IsOverlappingCidr", func() {
+	When("There are no base CIDRs", func() {
+		overlapping, err := IsOverlappingCIDR([]string{}, "10.10.10.0/24")
+		It("Should not return error", func() {
+			Expect(err).ToNot(HaveOccurred())
+		})
+		It("Should return false", func() {
+			Expect(overlapping).To(BeFalse())
+		})
+	})
+
+	When("At least one Base CIDR is superset of new CIDR", func() {
+		overlapping, err := IsOverlappingCIDR([]string{"10.10.0.0/16", "10.20.30.0/24"}, "10.10.10.0/24")
+		It("Should not return error", func() {
+			Expect(err).ToNot(HaveOccurred())
+		})
+		It("Should return true", func() {
+			Expect(overlapping).To(BeTrue())
+		})
+	})
+
+	When("At least one Base CIDR is subset of new CIDR", func() {
+		overlapping, err := IsOverlappingCIDR([]string{"10.10.10.0/24", "10.10.20.0/24"}, "10.10.30.0/16")
+		It("Should not return error", func() {
+			Expect(err).ToNot(HaveOccurred())
+		})
+		It("Should return true", func() {
+			Expect(overlapping).To(BeTrue())
+		})
+	})
+
+	When("At least one Base CIDR is subset of new CIDR", func() {
+		overlapping, err := IsOverlappingCIDR([]string{"10.10.0.0/16", "10.20.30.0/24"}, "10.10.10.0/24")
+		It("Should not return error", func() {
+			Expect(err).ToNot(HaveOccurred())
+		})
+		It("Should return true", func() {
+			Expect(overlapping).To(BeTrue())
+		})
```

Comment:
Looks like you missed removing this test/updating it. Its same as test case at L36

Reply:
yeah, missed cleanup after copy paste.